### PR TITLE
Mark as paid with in invoice

### PR DESCRIPTION
**Pull request by abinashpa**
## Notion card
https://www.notion.so/saeloun/Admin-and-Owner-should-be-able-to-mark-the-invoice-as-paid-from-invoice-287da7b196384920b5fae004646b9284

## Summary
- Admin or owner can now mark an invoice as paid from the invoice.

## Preview
https://www.loom.com/share/4aee1bef1ee74ecc98ea502cbc451c28

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor (non-breaking and retains the same functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [x] I have annotated changes in the PR that are relevant to the reviewer
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/controllers/internal_api/v1/invoices_controller.rb [hunk 1]
```diff
@@ -86,9 +85,7 @@ def invoice
     end
 
     def invoice_params
-      params.require(:invoice).permit(
-        policy(Invoice).permitted_attributes
-      )
+      params.require(:invoice).permit(InvoicePolicy.permitted_attributes)
```
Comment (prasanthchaduvula) [inline app/controllers/internal_api/v1/invoices_controller.rb:88]: Revert this change as the whole app follows this standard way `params.require(:invoice).permit(policy(Invoice).permitted_attributes)`

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

**Review by prasanthchaduvula — APPROVED** [review]

**Review by apoorv1316 — CHANGES_REQUESTED** [review]
@abinashpa Please check loom video https://www.loom.com/share/52de773e56d64da1ab26392f6ee7e537

**Comment by abinashpa** [comment]
@apoorv1316 The 3 dot option is not available on new invoice.
Screenshot 2022-12-26 at 9 28 16 PM

**Comment by apoorv1316** [comment]
@abinashpa As per the ticket "mark invoice as paid" option should be there for the overdue invoice which is currently missing. Can you please check?
Screenshot 2023-01-02 at 12 31 22 PM

**Comment by abinashpa** [comment]
@apoorv1316 fixed. 
Screenshot 2023-01-02 at 12 45 20 PM

**Review by apoorv1316 — APPROVED** [review]

**Pull request merged**